app/assets/javascripts/components/user_profiles/by_students_stats.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+const ByStudentsStats = React.createClass({
+
+  propTypes: {
+    username: React.PropTypes.string,
+    stats: React.PropTypes.object
+  },
+  render() {
+    return (
+      <div>
+        <h5>
+          {I18n.t('user_profiles.instructors_student_impact', { username: this.props.username })}
+        </h5>
+        <div className= "stat-display">
+          <div className= "stat-display__stat">
+            <div className="stat-display__value">
+              {this.props.stats.word_count}
```

Comment:
Should I still change the word_count keyword to word_count_by_students?( i.e after making the changes in the stats json view)